app/src/main/java/com/example/whattowatch/fragments/AccountFragment.java

```diff
@@ -42,11 +64,40 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         super.onViewCreated(view, savedInstanceState);
         ParseUser user = ParseUser.getCurrentUser();
 
-        etUsername = view.findViewById(R.id.etUsername);
+        tvUsername = view.findViewById(R.id.tvUsername);
         ivProfilePhoto = view.findViewById(R.id.ivProfilePhoto);
+        btnSignOut = view.findViewById(R.id.btnSignOut);
 
-        etUsername.setText(user.getUsername());
+        tvUsername.setText(user.getUsername());
+        ParseFile photo = user.getParseFile("profilePhoto");
```

Comment:
Is `profilePhoto` the indicator set in up in your Parse model? if yes use a global constant defined where the data model is defined